### DISCUSSION TITLE
Screen Capture shouldn't show "Tab"

### DISCUSSION THREAD

**Discussion by lsyfg** [MicrosoftEdge/WebView2Feedback#4155] [Q&A] [answered]
Screen Capture API shouldn't show "Tab"

**Comment by victorhuangwq** [comment]
This behavior should be fixed in 116 as per https://github.com/MicrosoftEdge/WebView2Feedback/issues/3419

**Reply by lsyfg** [reply]
Hi, I still can't understand. Is there a corresponding example? Thank you

**Reply by lsyfg** [reply]
How do I set hidden tabs？

**Reply by victorhuangwq** [reply]
What is the WebView2 runtime that you are using? I just want to make sure that you aren't using an older runtime.

**Reply by lsyfg** [reply]
The current webview2 runtime version is 1.0.2151.40, which is the latest version. I have seen the new version of instinctive settings, but I searched for them and don't know how to set them

**Reply by victorhuangwq** [reply]
> 1.0.2151.40

that is the SDK version. What about the WebView2 runtime version? It should be like `119.0.2151.58`

**Reply by lsyfg** [reply]
Sorry, I made a mistake. My application runtime version is 119.0.2151.44

**Reply by lsyfg** [reply, marked as answer]
Thank you for your prompt. I think I made a mistake in the runtime version. The latest version no longer has tabs by default. I think my problem has been resolved

**Reply by lsyfg** [reply]
The new version of the runtime can hide tabs, but I found that the screen sharing window displayed in the latest version is not complete. However, when I have multiple monitors, the version is 119.0.2151.58, and a lower version 118.0.2088.76 displays normally. Please refer to the screenshot I provided
version is 119.0.2151.58：
![QQ截图20231114133503](https://github.com/MicrosoftEdge/WebView2Feedback/assets/6143343/4f65e154-0ac0-415a-8b30-35f74a19fbd2)
version is 118.0.2088.76：
[
![QQ截图20231114141642](https://github.com/MicrosoftEdge/WebView2Feedback/assets/6143343/6ca1f5eb-214b-4e63-a1bb-3ea9b8294521)
](url)

**Reply by victorhuangwq** [reply]
thank you for the screenshot. I am assuming that you are trying to report that bug here? https://github.com/MicrosoftEdge/WebView2Feedback/issues/4160

Could you follow the [bug report form](https://github.com/MicrosoftEdge/WebView2Feedback/issues/new?assignees=&labels=bug&projects=&template=Bug+report+form.yml&title=%5BProblem%2FBug%5D%3A+) to create an issue? As it has all the information that we need to help diagnose the issue

**Reply by lsyfg** [reply]
I am very happy to provide it. I have submitted the problem information in the error report form。
I have closed Chapter 4160 because it has been resolved in version 118.0.2088.76.
But I have new questions and I hope to get your help. I use the WPF application to load webview2, and I fix the window size, but the displayed screen sharing window cannot change its size or coordinate position. Because it always appears on the right side of the WPF window and cannot be left aligned
